src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.colors.Color;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(Color color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public int getFirstLeg() {
+        return firstLeg;
+    }
+
+    public int getSecondLeg() {
+        return secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (float) (firstLeg * secondLeg) / 2;
```

Comment:
Optional: the are casting to float but returning type is double. It is a bit confusing and will be better if casting and return type will be the same